vagrant/provisioning/roles/saml-config/tasks/main.yml

```diff
@@ -52,20 +52,21 @@
     replace: "{{ item.replace }}"
   loop:
     - regexp: 'arkcase/acm/private/saml-keystore'
-      replace: 'arkcase/acm/private/keystore'
+      replace: 'arkcase/acm/private/arkcase.ks'
     - regexp: 'type="java.lang.String" value="password"'
-      replace: 'type="java.lang.String" value="changeit"'
+      replace: 'type="java.lang.String" value="{{ internal_host }}"'
     - regexp: 'key="armedia" value="AcMd3v\$"'
-      replace: 'key="eeoc-pre-prod-app-server" value="changeit"'
+      replace: 'key="{{ internal_host }}" value="{{ internal_host }}"'
     - regexp: 'type="java.lang.String" value="armedia"'
-      replace: 'type="java.lang.String" value="eeoc-pre-prod-app-server"'
+      replace: 'type="java.lang.String" value="{{ internal_host }}"'
 
-- name: copy the saml keystore
+- name: configure logout URL for SAML
   become: yes
   become_user: arkcase
-  copy:
+  replace:
```

Comment:
Since we don't need a special SAML keystore, we don't need to copy it.  But we do need to override the logout URL.